<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetadataPanel</class>
 <widget class="QWidget" name="MetadataPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Lighting Level:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="lighting_combo"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Background ID:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="background_id_edit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Sequence Number:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="sequence_number_edit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="lock_checkbox">
     <property name="text">
      <string>Lock Metadata</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="capture_button">
     <property name="text">
      <string>Capture</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
